change-management/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Change management
+permalink: /change-management/
+---
+
+* TOC
+{:toc}
+
+Keboola Connection already allows change management by the means
+of [configuration versioning](/components/#configuration-versions). That allows you to roll back any changes to any
+configuration. However, this only works on one configuration at a time. That does not help you when you need to test
+multiple dependent changes to the data pipeline. This is now solved by **development branches functionality**.
+
+{% include branches-beta-warning.html %}
+
+## What are development branches?
+
+Development branches in Keboola Connection allow you to make changes to multiple related configurations in isolation without
+affecting your production data pipelines. Also, component running in development branch writes data
+to [storage](/storage) uses a separate bucket. Your production data is not overwritten or changed in any way. You can propagate the changes
+from your development branch to production when you complete and test them.
+
+### Components
+
+When you create a development branch, you have all your component configurations available in that branch. When you
+update a configuration, new version is created in the branch, but the production version of the configuration remains
+the same.
+
+### Storage
+
+#### Write
+
+When a component in development branch writes data to storage, the bucket name stored in mapping is dynamically changed by prefixing the branche's internal ID. That means that your production bucket will not be overwritten.
```

Comment:
Vrátit link na transformace

Reply:
Protože ten content je fajn a kdyby se to přepsalo, tak bude vidět, že je to odsaď zalinkované.